submit_pubs_v2.py

```diff
@@ -24,19 +26,52 @@ def submit_pubs(new_osti_pubs, osti_creds, submission_limit):
         osti_pub['response_json'] = response.json()
 
         if response.status_code >= 300:
-            print("Response status code > 300...\n")
+            print("Submission Failure:")
+            print(response.json())
             osti_pub['response_success'] = False
 
-            # for Testing
-            pprint(osti_pub['submission_json'])
-            print(response)
-            pprint(response.json())
-
         else:
-            print("Submission OK.\n")
+            print("Submission OK.")
             osti_pub['response_success'] = True
             osti_pub['osti_id'] = osti_pub['response_json']['osti_id']
 
+            # If metadata submission was successful, submit the PDF
+            media_response = submit_media(osti_creds, osti_pub)
+            if media_response is False:
```

Comment:
How would the failed media submissions be retried?

Reply:
We have a few options -- can discuss at 1:1